__tests__/app.test.js

```diff
@@ -0,0 +1,35 @@
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const db = require("../db/connection");
+const data = require("../db/data/test-data");
+const request = require("supertest");
+
+afterAll(() => {
+  return db.end();
+});
+
+beforeEach(() => {
+  return seed(data);
+});
+
+describe("app testing", () => {
+  describe("/api/topics", () => {
+    test("GET 200: returns with a status of 200 and an array", async () => {
```

Comment:
Keep an eye on these test descriptions, you're stating here that the endpoint comes back with a 200 and an array but are only expecting a 200. Then later stating it comes back with an array. Could these be combined into one test?